lib/src/operance_data_column_width.dart

```diff
@@ -1,39 +1,34 @@
 // 🐦 Flutter imports:
-import 'package:flutter/widgets.dart';
```

Comment:
nice catch removing the unused import! 
Also remove the comment above since it's no longer required too